squid-blockexplorer/README.md

```diff
@@ -1 +1,37 @@
-# Squid for tracking account balances
+# Subspace block explorer squid
+
+Squid used to process and transform data for Subspace block explorer, based on [Squid template](https://github.com/subsquid/squid-template).
+
+## How to start
+
+```bash
+# 1. Install dependencies
+npm ci
+
+# 2. Compile typescript files
+make build
+
+# 3. Start target Postgres database and detach
+make up
+
+# 4. Start the processor
+make process
+```
+
+Make sure you provide following environmental variables:
+- `CHAIN_RPC_ENDPOINT` - Public node RPC endpoint
+- `ARCHIVE_ENDPOINT` - GraphQL endpoint to a local or remote archive
+> Please, refer to Subsquid [docs](https://docs.subsquid.io/archives/) to learn, more about Squid Archives
+
+In order to use GraphiQL explorer, run following in a separate terminal:
```

Comment:
```suggestion
In order to use GraphQL explorer, run following in a separate terminal:
```

Reply:
This is not a typo, I meant this:
<img width="189" alt="image" src="https://user-images.githubusercontent.com/13568875/201461278-20de02f7-5eaf-40b7-85d0-329060317b2d.png">

Reply:
My bad lol